### PR TITLE
Update plugins + release bom every time we release something

### PR DESCRIPTION
**Pull request by kleewho**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -119,14 +118,12 @@ task cucumber(type: Test) {
...
-        includeNonPublic = false
-        skipEmptyPackages = true
-    }
-}
+//tasks.dokkaGfm.configure {
+//    dokkaSourceSets {
+//        includeNonPublic.set(false)
+//        skipEmptyPackages.set(true)
+//    }
+//}
```
Comment (kleewho) [inline build.gradle:126]: I didn't know how exactly configure it the same

**Comment by parfeon** [comment]
@kleewho bom will be released with proper version?

**Comment by parfeon** [comment]
@kleewho i don’t understand how it will work. Each component will be published along with BOM (version for it maybe the same through release of components under different versions, right?)

**Comment by kleewho** [comment]
I have to rethink it

**Review by Chesteer89 — APPROVED** [review]

**Review by marcin-cebo — APPROVED** [review]

**Pull request merged**